nilearn/plotting/html_stat_map.py

```diff
@@ -117,7 +117,7 @@ def _threshold_data(data, threshold=None):
 
 
 def _save_sprite(
-    data, output_sprite, vmax, vmin, mask=None, cmap="Greys", format="png"
+    data, output_sprite, vmax, vmin, mask=None, cmap="Grays", format="png"
```

Comment:
double check if 'Grays' is a valid color

Reply:
Might be related to https://github.com/matplotlib/matplotlib/pull/28116. This is a recent fix that hasn't been released yet. It will be part of the upcoming v3.10.

Reply:
yeah I remembered something like this

Temped to put in the ignore list of codespell untill then.